internal/support/scheduler/v2/application/scheduler/manager.go

```diff
@@ -0,0 +1,131 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package scheduler
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/utils"
+	"github.com/edgexfoundry/edgex-go/internal/support/scheduler/config"
+	"github.com/edgexfoundry/edgex-go/internal/support/scheduler/v2/infrastructure/interfaces"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/logger"
+
+	queueV1 "gopkg.in/eapache/queue.v1"
+)
+
+type manager struct {
+	ticker                *time.Ticker
+	lc                    logger.LoggingClient
+	config                *config.ConfigurationStruct
+	mutex                 sync.Mutex
+	executorQueue         *queueV1.Queue
+	intervalToExecutorMap map[string]*Executor
+	actionToIntervalMap   map[string]string
+}
+
+func NewManager(lc logger.LoggingClient, config *config.ConfigurationStruct) interfaces.SchedulerManager {
+	return &manager{
+		ticker:                time.NewTicker(time.Duration(config.Writable.ScheduleIntervalTime) * time.Millisecond),
```

Comment:
ScheduleIntervalTime should not be under Writable, because it won't be changed at runtime

Reply:
Moved the ScheduleIntervalTime to the root of the config.